Library/Navigation.swift

```diff
@@ -400,6 +401,18 @@ private func pledgeNew(_ params: RouteParamsDecoded) -> Navigation? {
   return nil
 }
 
+private func projectRedeem(_ params: RouteParamsDecoded) -> Navigation? {
```

Comment:
Same here; since we handle all backings urls in the "survey webview", we should treat them all as `projectSurvey` urls. (The webview itself checks if a url is of type `projectSurvey` and only allows it to be loaded if it is.)

Reply:
so ":backing_tab" and "redeem" result in the same url param?